### PR TITLE
enhance docs for msk

### PR DESCRIPTION
**Pull request by HarshCasper**

### CODE REVIEW

**Comment by MarcelStranak** [comment]
We should add the new details that came up during the [customer issue](https://localstack-community.slack.com/archives/C05HWLVCU8J/p1690199333428459).

File: content/en/user-guide/aws/managed-streaming-for-kafka/index.md [hunk 1]
```diff
@@ -1,44 +1,52 @@
 ---
 title: "Managed Streaming for Kafka (MSK)"
 linkTitle: "Managed Streaming for Kafka (MSK)"
-categories: ["LocalStack Pro"]
 description: >
   Get started with AWS Managed Streaming for Kafka (MSK) on LocalStack
-aliases:
-  - /aws/managed-streaming-for-kafka/
```
Comment (alexrashed) [inline content/en/user-guide/aws/managed-streaming-for-kafka/index.md:8]: Removing the alias means that old links might break, doesn't it?

File: content/en/user-guide/aws/managed-streaming-for-kafka/index.md [hunk 2]
```diff
@@ -82,41 +93,48 @@ The expected output is something like the following
...
 
-In this step of using LocalStack MSK, we'll download and use the Kafka command line interface (CLI) to create a topic that produces and consumes data.
+To use LocalStack MSK, you can download and utilize the Kafka command line interface (CLI) to create a topic for producing and consuming data.
 
-Run the following command to download Apache Kafka.
+To download Apache Kafka, execute the following command.
 
 {{< command >}}
 wget https://archive.apache.org/dist/kafka/2.2.1/kafka_2.12-2.2.1.tgz
 tar -xzf kafka_2.12-2.2.1.tgz
```
Comment (alexrashed) [inline content/en/user-guide/aws/managed-streaming-for-kafka/index.md:104]:
```suggestion
$ wget https://archive.apache.org/dist/kafka/2.2.1/kafka_2.12-2.2.1.tgz
$ tar -xzf kafka_2.12-2.2.1.tgz
```

File: content/en/user-guide/aws/managed-streaming-for-kafka/index.md [hunk 3]
```diff
@@ -202,19 +224,20 @@ The following response is to be expected
...
+
+You can use the `kafka-console-producer.sh` client script to publish messages to the topic. By doing so, you can closely monitor the execution of Lambda functions within Docker containers as new messages arrive by simply observing the LocalStack log output.
 
 ## Delete the local MSK cluster
 
-Run the following command to list your MSK clusters
+You can delete the local MSK cluster using the [`DeleteCluster`](https://docs.aws.amazon.com/cli/latest/reference/kafka/delete-cluster.html) API. To do so, you must first obtain the ARN of the cluster you want to delete. Run the following command to list all the clusters in the region:
 
 {{< command >}}
 awslocal kafka list-clusters --region us-east-1
```
Comment (alexrashed) [inline content/en/user-guide/aws/managed-streaming-for-kafka/index.md:236]:
```suggestion
$ awslocal kafka list-clusters --region us-east-1
```

File: content/en/user-guide/aws/managed-streaming-for-kafka/index.md [hunk 4]
```diff
@@ -126,53 +144,57 @@ The following step is optional and may not be required, depending on the operati
...
 
 ### Adding a local MSK trigger 
 
-The following example uses the Lambda Event Source Mapping API to map a Lambda function named my-kafka-function to a Kafka topic named LocalMSKTopic. The topic's starting position is set to LATEST.
+You can add a Lambda Event Source Mapping API to create a mapping between a Lambda function, named `my-kafka-function`, and a Kafka topic called `LocalMSKTopic`. The configuration for this mapping sets the starting position of the topic to `LATEST`.
+
+Run the following command to use the [`CreateEventSourceMapping`](https://docs.aws.amazon.com/lambda/latest/dg/API_CreateEventSourceMapping.html) API by specifying the Event Source ARN, the topic name, the starting position, and the Lambda function name.
 
 {{< command >}}
 awslocal lambda create-event-source-mapping \
```
Comment (alexrashed) [inline content/en/user-guide/aws/managed-streaming-for-kafka/index.md:200]:
```suggestion
$ awslocal lambda create-event-source-mapping \
```

File: content/en/user-guide/aws/managed-streaming-for-kafka/index.md [hunk 5]
```diff
@@ -82,41 +93,48 @@ The expected output is something like the following
...
 {{< / command >}}
 
-## Create a kafka topic
+### Create a Kafka topic
 
-In this step of using LocalStack MSK, we'll download and use the Kafka command line interface (CLI) to create a topic that produces and consumes data.
+To use LocalStack MSK, you can download and utilize the Kafka command line interface (CLI) to create a topic for producing and consuming data.
 
-Run the following command to download Apache Kafka.
+To download Apache Kafka, execute the following command.
```
Comment (alexrashed) [inline content/en/user-guide/aws/managed-streaming-for-kafka/index.md:100]:
```suggestion
To download Apache Kafka, execute the following commands.
```

**Review by alexrashed — COMMENTED** [review]
Awesome! This is a great set of changes to refresh the MSK docs! I only found a few nitpicks, afterwards we're good to merge! 🚀

**Pull request merged**